proxy-recorder/launch-gatling-proxy.sh

```diff
@@ -169,6 +169,26 @@ read
 
 FIND_COMMAND='find ./user-files/bodies -name *.txt'
 FIND_COUNT=`${FIND_COMMAND} |wc -l`
+  
+# The find count will return the number of things found, which will
+# be greater than 1 when there are "leftover" results in the directories
+# due to past aborts (like with ^C).  Issue a warning about needing
+# a little house cleaning, then try to recover and continue on 
+#
+if [ ${FIND_COUNT} -gt 1 ]	# Use numeric comparison, not string
+then
+   echo "
+OOPS!
+There is left-over recording data in the \"./user-files/bodies\" directory 
+and the \"../simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/\"
+directory (.txt and .scala files).  If the recording information for them is complete, 
+then you can manually copy the existing files to the proper directories or delete them 
+and re-run this recording if necessary.
+Exiting.
```

Comment:
Is the intention of this code now to exit when this happens or continue on?  The comment above the `if` statement makes it sound like the script will try to continue on but the error message makes it sound like it is going to exit.

Assuming the intention is to exit, it looks like the `exit 1` line needs to be outside of the ending double quote.  

Also, in an earlier comment, it sounded like you were going to look into putting the check for leftover content before where the Gatling proxy recorder is launched but it appears that this check would still be done after the new recording has finished.  Are you still planning on moving this check to earlier in the script?